javascripts/discourse/connectors/below-site-header/maintenance-message.js

```diff
@@ -0,0 +1,10 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+
+export default class MaintenanceMessage extends Component {
+  @service currentUser;
+
+  get messageVisible() {
+    return !this.currentUser || !this.currentUser.admin;
```

Comment:
This is incredibly difficult to read. I think it works out as 'show to all non-admins'? This feels clearer:
```suggestion
    return !this.currentUser?.admin;
```